src/components/FlowRunTimelineTaskDetails.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <template v-if="task">
+    <page-heading :crumbs="crumbs" size="sm" class="mb-2">
+      <template #actions>
+        <slot />
+      </template>
+    </page-heading>
+    <div class="timeline-task-details__content">
+      <p-key-value label="State" :alternate="alternate">
+        <template #value>
+          <StateBadge :state="task.state" class="timeline-task-details__state-badge" />
```

Comment:
```suggestion
          <StateBadge :state="task.state" class="flow-run-timeline-task-details__state-badge" />
```
BEM says this should contain the full name of the component